components/sysutils/bacula/Makefile

```diff
@@ -68,7 +68,6 @@ REQUIRED_PACKAGES += database/mariadb-103
 REQUIRED_PACKAGES += database/mariadb-103/client
 
```

Comment:
You'll need to add database/postgres-12/developer as a manually added build dependency.

Reply:
added